src/styles/blocks/header.scss

```diff
@@ -0,0 +1,99 @@
+* {
+  padding: 0;
+  margin: 0;
+  list-style: none;
+  box-sizing: border-box;
+}
+
+a {
+  text-decoration: none;
+  color: inherit;
+}
+
+html {
+  font-size: 16px;
+  font-family: Roboto, sans-serif;
+  font-style: normal;
+  font-weight: 500;
+}
+
+:root {
+  --header-height: 60px;
+}
+
+.roboto-medium {
+  font-family: Roboto, sans-serif;
+  font-weight: 500;
+  font-style: normal;
+}
+
+.header {
+  display: flex;
+  align-items: center;
+  margin: 0 auto;
+  padding: 0 50px;
+  justify-content: space-between;
+  width: 100%;
+}
+
+.logo {
+  display: block;
+  height: 40px;
+  width: 40px;
+}
+
+.nav {
+  text-transform: uppercase;
+}
+
+.nav__list {
+  display: flex;
+}
+
+.nav__item {
+  color: black;
+  transition: all ease-in-out 0.15s;
```

Comment:
The `color` property for `.nav__item` seems to be redundant since `.nav__link` (which is a child element) has its own `color` property set. You can remove the `color` property from `.nav__item` to simplify the styles.